README.md

```diff
@@ -165,6 +165,14 @@ get duplicated chunks on the page. Be especially careful if you might be calling
 <%= stylesheet_pack_tag 'map' %>
 ```
 
+For instance:
```

Comment:
```suggestion
For instance let's say you only want to include a `map` pack when a specific `my_condition` method is truthy, 
you can define an array of default packs which includes you `calendar` pack. 

Then you can call the `javascript_pack_tag` helper once with the Ruby array splat operator so that each string 
in the `packs` array is sent as an individual argument:
```